src/main/java/com/conveyal/datatools/common/status/MonitorableJob.java

```diff
@@ -93,7 +94,17 @@ private void registerJob() {
         if (userJobs == null) userJobs = Sets.newConcurrentHashSet();
         userJobs.add(this);
 
-        DataManager.userJobsMap.put(this.owner, userJobs);
+        DataManager.userJobsMap.put(retrieveUserId(), userJobs);
+    }
+
+    @JsonProperty("owner")
+    public String retrieveUserId() {
+        return this.owner == null ? "unknown" : this.owner.getUser_id();
```

Comment:
Should we allow the owner to be null? Seems like a job should always have information on who the owner is.